glean_parser/swift.py

```diff
@@ -51,6 +51,8 @@ def iterencode(self, value):
                     yield from self.iterencode(subvalue)
                     first = False
                 yield "]"
+            elif value is None:
+                yield "nil"
```

Comment:
Interesting.  I wonder why we don't hit this case in Kotlin (since it presumably also might have some `None`'s that would need to become `null`s)...

Reply:
That is ... actually ... I don't know.
In my first tests I didn't run into this either, but with the file from Glean it happened.

Let me double-check this before we merge.

Reply:
Ah! Because this is all based on `JSONEncoder`, which encodes _nothing_ as the JSON value `null`, which then is also absolutely valid Kotlin code.

In Swift it's `nil` though, so we have to explicitly change it.